frontend/src/app/shared/icons.ts

```diff
@@ -18,6 +18,8 @@ export const actionIcons = {
     expand: 'caret-down',
     collapse: 'caret-up',
     edit: 'pencil',
+    up: 'chevron-up',
+    down: 'chevron-down',
```

Comment:
Hm, this might look too similar to the `collapse`/`expand` icons, which appear in a similar position. Perhaps `'arrow-up'` / `'arrow-down'` would be more distinctive?